docs/dogfood/index.md

```diff
@@ -0,0 +1,143 @@
+# Dogfooding
+
+This guide explains how to [dogfood](https://www.techopedia.com/definition/30784/dogfooding) coder for employees at Coder.
+
+## How to 
+
+The following explains how to do certain things related to dogfooding.
+
+### Dogfood using Coder's Deployment
+
+1. Go to [https://dev.coder.com](https://dev.coder.com/) 
+    - If you see a dialog/pop-up, hit "Cancel"
+2. Sign in with GitHub
+3. Clone the repo: `git clone git@github.com:coder/coder.git` 
+4. Run `make bin`. The binaries will be created in the `./dist` folder.
+    
+    <aside>
+    💡 We usually rename the `coder` binary to `coderv2` so it doesn’t clash with the old version!
```

Comment:
I don't think we should suggest this anymore. v1 devs should create a v1 workspace from my perspective.